hydra/app/views/catalog/_pdf.html.erb

```diff
@@ -1,19 +1,19 @@
 <%# Ruby Page Variables %>
-<% identifier = @document[:id] %>
+<% id = @document[:id] %>
 <% project = @document[:project_ssi] %>
 <% num_files = @document[:num_files_isi] %>
 <% description = @document[:description_tesim] %>
 <% preview = @document[:preview_ssi] %>
 
 <div class="row">
-  <div class="col-md-4">      
+  <div class="col-md-4">
       <a class="document-image-full" href="<%= preview %>">
-        <% if Acda.where(identifier: identifier).first&.thumbnail_file.nil? %>
+        <% if @thumbnail_file.nil? %>
           <span class="fa fa-file-pdf-o fa-5x" aria-hidden="true"></span>
         <% else %>
-          <%= image_tag("/image/#{identifier}", title:"#{@document[:title_ssi]}", alt:description, class:"full-size-responsive") %> 
+          <%= image_tag("/image/#{id}", title:"#{@document[:title_ssi]}", alt:description, class:"full-size-responsive") %>
```

Comment:
Again curious about the rename.

Reply:
i'm curious why not rename it? if it's not an identifier and it is an id, why keep calling it an identifier?